packages/gatsby-recipes/foo

```diff
@@ -0,0 +1,7 @@
+set -euxo pipefail
+IFS=$'\n\t'
+
+false | cat
+false
+
+echo ok
```

Comment:
Is this file meant to go in?

Reply:
Woopsie! Nope, was testing something unrelated 😅